packages/lib-classifier/src/plugins/tasks/DrawingTask/models/DrawingTask.js

```diff
@@ -48,8 +48,8 @@ export const Drawing = types.model('Drawing', {
       })
     },
 
-    get isComplete () {
-      return self.tools.reduce((isTaskComplete, tool) => isTaskComplete && tool.isComplete, true)
+    isComplete (annotation) {
+      return self.tools.reduce((isTaskComplete, tool) => isTaskComplete && tool.isComplete, true) && !!annotation?.isComplete
```

Comment:
`annotation.isComplete` is always true for drawing annotations, so I think you don't need to check it here.